### PR TITLE
(CDAP-8701) Enlarge the local log buffer queue size

### PR DESCRIPTION
**Pull request by chtyim**
- Also determine without blocking if a log event is coming from log processor itself and ignores it.

### CODE REVIEW

File: cdap-standalone/src/main/resources/logback.xml [hunk 1]
```diff
@@ -128,7 +128,8 @@
     <appender-ref ref="EXTERNAL_AUTH_AUDIT" />
   </logger>
 
-  <root level="DEBUG">
+  <!--<root level="DEBUG">-->
+  <root level="TRACE">
```
Comment (anwar6953) [inline cdap-standalone/src/main/resources/logback.xml]: remove this change.

Comment (chtyim) [reply cdap-standalone/src/main/resources/logback.xml]: ouch.

**Comment by anwar6953** [comment]
LGTM

**Comment by chtyim** [comment]
Building https://builds.cask.co/browse/CDAP-DUT5860

**Comment by chtyim** [comment]
Build passed. Squashed and merging.

**Pull request merged**